api/account.go

```diff
@@ -10,6 +10,10 @@ import (
 )
 
 const (
+	// API Keycloak out-of-the-box
+	ctAccountLinkedAccountsPath         = "/auth/realms/:realm/account/linked-accounts"
```

Comment:
Rename kcAccountLinkedAccountsPath (kc prefix for Keycloak API, ct prefix for Cloudtrust API)